src/components/FeatureViews/Accounts/TabComponents/CourseSessionStatus.js

```diff
@@ -262,6 +266,15 @@ const CourseSessionStatus = () => {
 
     };
 
+    const closeUnenrollDialog = (toUnenroll) => event => {
+        event.preventDefault();
+        setUnenrollWarningOpen(false);
+        if(toUnenroll){
+            api.deleteEnrollment(courseID, studentID, enrollment.enrollment_id);
```

Comment:
Can you make this action take in the whole enrollment and have it extract course and student inside? It would make calling this later on simpler.